### ISSUE TITLE
ToggleSmall fails IBM a11y checkpoint 1.4.1

### ISSUE THREAD

**Issue by matthew-chirgwin** [carbon-design-system/carbon#3159]
## Environment

> Operating system: OSX

> Browser: Tested on Firefox, likely all browsers

## Detailed description

> What version of the Carbon Design System are you using?

```    
"carbon-components": "9.87.2",
"carbon-components-react": "6.113.1",
"carbon-icons": "7.0.7",
```

But have also seen similar issues on the latest storybook for Carbon X.

> What did you expect to happen/What happened instead?

Checkpoint 1.4.1 asserts that colour alone cannot be used to convey information. In manual AVT testing, we have found that the ToggleSmall component in both Carbon 9 and X fails this checkpoint when deselected. As an example, the below screenshot is of the Carbon 9 case:

Deselected:

![Screenshot 2019-06-24 at 10 35 52](https://user-images.githubusercontent.com/43138971/60008449-0cfc3380-966c-11e9-91ca-e464b2833d0c.png)

Selected:

![Screenshot 2019-06-24 at 10 36 48](https://user-images.githubusercontent.com/43138971/60008473-15546e80-966c-11e9-855d-f8a745bc8260.png)

In 'normal' cases, deselection is flagged by a grey blob - but as seen here, this is not visible when colour is disabled in the browser (see steps below to enable this behaviour).

> What WCAG 2.1 checkpoint does the issue violate?

See https://www.ibm.com/able/guidelines/ci162/color.html and https://www.w3.org/WAI/WCAG21/Understanding/use-of-color

## Steps to reproduce the issue

Go to any instance of the ToggleSmall on Firefox.

On a mac, go to Firefox->Preference->Colors...

Set colours as follows:

![image](https://user-images.githubusercontent.com/43138971/60008912-d246cb00-966c-11e9-8c31-8bc2f9e52868.png)

## Additional information

If possible, could any fix please be released for Carbon 9 as well as latest.

**Comment by matthew-chirgwin** [comment]
Bump - this is still an issue

**Comment by tw15egan** [comment]
Seems to still be an issue.

**Comment by joshblack** [comment]
Hey @matthew-chirgwin! I believe this is related to this comment: https://github.com/carbon-design-system/carbon/issues/4783#issuecomment-567255561 copying the relevant bits over:

> We don't support this method of high contrast color inversion on Firefox in macOS.
>
> If a user wants high contrast support for Firefox in macOS they should navigate to the Accessibility options screen and check one or both of options Invert Colors and Increase Contrast.

Hope this helps!

**Comment by matthew-chirgwin** [comment]
Thanks @joshblack for following up! If needed I will re open this if our testers still encounter the issue following the above.
